content/en/docs/getting-started/installation.md

```diff
@@ -11,190 +11,302 @@ Falco can be used for Kubernetes runtime security.
 The most secure way to run Falco is to install Falco directly on the host system so that Falco is isolated from Kubernetes in the case of compromise.
 Then the Falco alerts can be consumed through read-only agents running in Kubernetes.
 
-You can also run Falco directly in Kubernetes as a daemonset using Helm, see the [third party integrations](../third-party)
+You can also run Falco directly in Kubernetes as a Daemonset using Helm, see the [third-party integrations](../third-party)
 {{% /pageinfo %}}
 
+There are 2 main ways to install Falco on your host:
 
-If Falco is installed using the package manager artifacts below, you will have the following in place:
+1. Falco packages (`.deb`, `.rpm`)
+2. Falco binary (`.tar.gz`)
 
- - Falco userspace program scheduled and watched via `systemd`
- - Falco driver installed via the package manager (either kernel module or eBPF depending on the host)
- - Sane and default configuration file installed in `/etc/falco`
+## Falco packages
 
-Alternatively, it is also possible to use a binary package as [explained below](#linux-binary).
+{{% pageinfo color="secondary" %}}
 
-## Installing
+The Falco packages shipped with `Falco 0.34` support for the first time other drivers besides the kernel module. The new Systemd units' names are:
```

Comment:
`systemd` is the right name, without the uppercase `S`

Reply:
ty!